### PR TITLE
Makes the Singularity/Tesla on Boxstation Randomized

### PR DESCRIPTION
**Pull request by redmoogle**
### Intent of your Pull Request

The Tesla/Sing is now randomized and you dont get both now

### Why is this good for the game?

Hmm yes forcing you to use the slightly more dangerous engine yes

#### Changelog

:cl:  
tweak: The Singularity/Tesla engine is now randomized on box
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
:thinking:

**Comment by athena413** [comment]
sure, lets see it. i'd like to see how it effects engineering

File: code/modules/power/randomengine.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/obj/machinery/the_singularitygen/random
+	name = "Random Engine" // Blatant code reuse shh
+	desc = "Tell Coders if you see this"
+	icon = 'icons/effects/landmarks_static.dmi'
+	icon_state = "generic_event" // Too Lazy to make sprit so use random questionmark box
+
+/obj/machinery/the_singularitygen/random/Initialize(mapload)
+	..()
+	switch(prob(50)) // 50% chance to trigger either
```
Comment (Bibby0110) [inline code/modules/power/randomengine.dm:9]: Kek, nice code.

Use an if statement, not a switch

**Comment by redmoogle** [comment]
To flex on bibby Im going to put more effort into a pr then he ever has

**Pull request closed without merging**